src/app/enums/product-type.enum.ts

```diff
@@ -4,3 +4,10 @@ export enum ProductType {
   Core = 'CORE',
   Enterprise = 'ENTERPRISE',
 }
+
+export const ProducTypetReadableText = {
```

Comment:
Typo in name.
Prefer `Map` for this with typing, see something like `nfsAclTagLabels` (translation is not necessary).